specs/account_spec.rb

```diff
@@ -137,35 +138,62 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Wave 2" do
+describe "Wave 2" do
   describe "Account.all" do
+
+    before do
+      @accounts = Bank::Account.all
+    end
     it "Returns an array of all accounts" do
-      # TODO: Your test code here!
-      # Useful checks might include:
-      #   - Account.all returns an array
-      #   - Everything in the array is an Account
-      #   - The number of accounts is correct
-      #   - The ID and balance of the first and last
-      #       accounts match what's in the CSV file
-      # Feel free to split this into multiple tests if needed
+      #new_account =  Bank::Account.all #- Account.all returns an array
+      @accounts.must_be_kind_of Array
+      @accounts.each do |account|
+        account.must_be_instance_of Bank::Account #Account class
+      end
+    end
+    it "The number of accounts is correct" do
+      #new_account =  Bank::Account.all
+      @accounts.length.must_equal 12 #(12)
+      #creating test need must xxx
     end
   end
 
   describe "Account.find" do
     it "Returns an account that exists" do
       # TODO: Your test code here!
+      acct_id_1212 =  Bank::Account.find(1212)
+      acct_id_1212.must_be_instance_of Bank::Account #Account class
     end
 
     it "Can find the first account from the CSV" do
       # TODO: Your test code here!
+      acct_id_1212 =  Bank::Account.find(1212)
+      acct_id_1212.must_be_instance_of Bank::Account #Account class
```

Comment:
In addition to checking that what you got back from `Account.find` is an account, you should verify that the ID matches what you asked for. As is, `find` could return any old account and your test would still pass.